### PR TITLE
Feature/appointment

### PR DESCRIPTION
**Pull request by tanorpanda**
## Overview

This Pull Request is for appointment integration with BE

### CODE REVIEW

File: src/pages/dashboard/appointment/detail.js [hunk 1]
```diff
@@ -37,48 +39,39 @@ export function DetailAppointment() {
...
-      meeting_id = '',
+      appointment_datetime = '',
+      appointment_location = '',
+      id = '',
       status = '',
-      supplier = {},
-      supplier_name = '',
-      time = '',
+      Supplier = {},
+      Agent = {},
```
Comment (dcbon) [inline src/pages/dashboard/appointment/detail.js:47]: sort

File: src/pages/dashboard/appointment/index.js [hunk 2]
```diff
@@ -29,12 +31,21 @@ export function Appointment() {
...
 
   const fetchAllAppointments = async () => {
-    //  TODO: hit BE api
-    setListAppointment(APPOINTMENT_MOCK_LIST);
+    const { data } = await AppointmentService.getAllAppointments({
+      keyword: parsed.keyword,
+      page: parsed.page,
+      start_date: parsed.start_date,
+      end_date: parsed.end_date,
+      status: parsed.type,
```
Comment (dcbon) [inline src/pages/dashboard/appointment/index.js:45]: sort

**Review by dcbon — APPROVED** [review]

File: src/pages/dashboard/appointment/index.js (hunk 2 above)
Comment (tanorpanda) [reply src/pages/dashboard/appointment/index.js]: Baik Mba

File: src/pages/dashboard/appointment/detail.js (hunk 1 above)
Comment (tanorpanda) [reply src/pages/dashboard/appointment/detail.js]: Baik Mba

**Pull request merged**